staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go

```diff
@@ -1211,7 +1237,7 @@ func TestGetListComponentStatus(t *testing.T) {
 	cmd.Run(cmd, []string{"componentstatuses"})
 
 	expected := `NAME            STATUS      MESSAGE   ERROR
-servergood      Healthy     ok        
+servergood      Healthy     ok
```

Comment:
Don't

Reply:
Dropping these spaces was originally added to 1.18.2 in https://github.com/openshift/kubernetes-kubectl/pull/21, should we re-add them?